src/google/Common.js

```diff
@@ -144,8 +145,14 @@
         google.visualization.events.addListener(this._chart, "select", function () {
             var selectedItem = context._chart.getSelection()[0];
             if (selectedItem) {
-                context.click(context.rowToObj(context.data()[selectedItem.row]), context.columns()[selectedItem.column]);
+                context._selection = {
+                    data: context.rowToObj(context.data()[selectedItem.row]),
+                    column: context.columns()[selectedItem.column]
+                };
+            } else {
+                context._selection = {};
             }
+            context.click(context._selection.data, context._selection.column, Object.keys(context._selection).length !== 0 ? true : false);
```

Comment:
This will fail if selectedItem is null/undefined.
No need for the ```? true : false``` piece since the LHS is already evaluating to true/false.